src/main/java/GameList.java

```diff
@@ -115,5 +115,7 @@ public void displayGamesGivenStatus(Status status) {
     // ------GETTERS------
     public String getName() { return name; }
 
+    public List<Game> getGameList() { return gameList; }
```

Comment:
can we refactor this function as "getGames" that way we're not calling gb.getGameList().getGameList()...
gb.getGameList().getGames() seems more readable to me
```suggestion
    public List<Game> getGames() { return gameList; }
```